mods/voidmulti.sc2mod/base.sc2data/gamedata/weapondata.xml

```diff
@@ -99,11 +99,16 @@
         <Effect value="DisruptionBeam"/>
         <LegacyOptions index="CanRetargetWhileChanneling" value="1"/>
     </CWeaponLegacy>
+    <CWeaponLegacy id="FusionCutter">
+        <LegacyOptions index="NoDeceleration" value="1"/>
+        <AllowedMovement value="Slowing"/>
+    </CWeaponLegacy>
     <CWeaponLegacy id="GuassRifle">
         <MinScanRange value="5.5"/>
     </CWeaponLegacy>
     <CWeaponLegacy id="HydraliskMelee">
-        <Period value="0.75"/>
+        <Period value="0.825"/>
+        <DamagePoint value="0.14"/>
```

Comment:
Hydralisk melee attack's Damage point reduced from 0.21 to 0.1 seconds (from 0.3 to 0.14 on normal speed)
Hydralisk melee attack period raised from 0.54 to 0.59 to match the ranged attack's speed (from 0.75 to 0.825 on normal speed)

This is the melee attack. It was faster than the ranged attack which was not intended. Also, the damage points were different, so the damage was dealt at different amounts of time into the attack making the melee attack worse and maybe not deal damage when kiting melee units